planex/planex_handler.py

```diff
@@ -0,0 +1,65 @@
+from typing import TextIO
+import urllib.request
+import json
+from datetime import datetime
+from bs4 import BeautifulSoup
+
+
+DUMP_FILE_PLANEX = "planex.json"
+PLANEX_URL = "http://planex.insa-toulouse.fr/"
+# 10 min
+CUSTOM_MESSAGE_INTERVAL = 10 * 60 * 1000
+
+
+def parse_html(html_string: str) -> BeautifulSoup:
+    return BeautifulSoup(html_string, "html.parser")
+
+def is_planex_down() -> bool:
+    try:
+        with urllib.request.urlopen(PLANEX_URL) as response:
+            if response.getcode() != 200:
```

Comment:
If Planex is not `200 OK`, shouldn't it mean something isn't working properly ? My intuition is that this should return `True`.

Reply:
fixed a logic error in proposed solution :+1: